<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat2</class>
 <widget class="QDialog" name="chat2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>31</y>
     <width>971</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="employe2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>69</y>
     <width>821</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="employe2_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
